core/src/main/java/org/elasticsearch/threadpool/ThreadPoolStats.java

```diff
@@ -188,6 +206,7 @@ public void writeTo(StreamOutput out) throws IOException {
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, ToXContent.Params params) throws IOException {
         builder.startObject(Fields.THREAD_POOL);
+        Collections.sort(stats);
```

Comment:
@bleskes OK - that's a very good point, thanks! I'll move it to a `stats()` method.